crates/orchestrator/src/lib.rs

```diff
@@ -248,6 +259,8 @@ where
     // Assumes nodes will set their own index that they received from the
     // 'identity' endpoint
     fn post_getconfig(&mut self, _node_index: u16) -> Result<NetworkConfig<KEY>, ServerError> {
+        self.manual_start_allowed = false;
```

Comment:
I don' t think we want this here.  Iirc, this function gets the initial config before keys have been registered.  (We actually don't really need this function at all and imo it just adds extra confusion, but that can be for another PR :) ).